### ISSUE TITLE
Operators of unary or binary expressions should be placed on same line as (right) operand

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#534]
We should file an issue if a binary expression such as `&&` or `||` spans multiple lines and the operator token is placed at the end of the line.

Following code should produce a warning:
```c#
if (a &&
    b)
{
    // something to do here
}
```

It should be changed to:
```c#
if (a
    && b)
{
    // something to do here
}
```

The reason is that:
1) It is easier to read because, when at the end, there is always the question of what comes next after the operator ( "and what?" ) which does not when placed on the next line.
2) Merging can be done more easily.
    It is more likely that - _when placed on the same line as the next operator_ - only a single line changed, while - _when placed on the on the line end_ - at least two lines have to be merged; resulting more likely in merge conflicts.

**Comment by RalfKoban** [comment]
We should also take the unary operators such as `!` into account and place them also on the same line as their operands.
